app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CustomerItemsController < ApplicationController
+  def create
+    item = Item.find(params[:item_id])
+    customer = Customer.find(params[:customer_id])
+    CustomerItem.create!(customer: customer, item: item)
+
```

Comment:
This looks great. If you wanted to do it in a single line it would look like:
```ruby
CustomerItem.create!(customer: params[:customer_id], item: params[:item_id])
```